extensions/wikia/AdEngine/js/video/googleIma.js

```diff
@@ -0,0 +1,76 @@
+/*global define, google*/
+define('ext.wikia.adEngine.video.googleIma', [
+	'ext.wikia.adEngine.video.vastBuilder',
+	'wikia.lazyqueue',
+	'ext.wikia.adEngine.utils.scriptLoader'
+], function (vastBuilder, lazyQueue, scriptLoader) {
+	'use strict';
+	var adDisplayContainer,
+		adsLoader,
+		adsManager,
+		imaLibraryUrl = '//imasdk.googleapis.com/js/sdkloader/ima3.js',
```

Comment:
It can't be served from our domain, inside this script is some kind of security function which check domain compatibility with current domain and forced to load it from google servers. (SP guys are probably modifying this file to serve it from different domain)